### PR TITLE
BLD: Fix uploading of aarch64 wheels

### PR DESCRIPTION
**Pull request by lithomas1**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Comment by mroeschke** [comment]
Nice, were you able to confirm a similar workflow works locally?

**Comment by lithomas1** [comment]
It should work(I tested all the way up to the actual uploading). 
Unfortunately, this is the sort of thing where you have to merge and see, so this might take a couple attempts.

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,15 +52,26 @@ jobs:
...
+
+            MAMBA_DIR="$HOME/miniconda3"
+            rm -rf $MAMBA_DIR
+            ./minimamba.sh -b -p $MAMBA_DIR
+
+            export PATH=$MAMBA_DIR/bin:$PATH
+
+            mamba install -y -c conda-forge anaconda-client
+            
+            anaconda -q -t ${TOKEN} upload --skip -u ${ANACONDA_ORG} ./wheelhouse/*.whl
```
Comment (lithomas1) [inline .circleci/config.yml:73]: Just testing that the anaconda command can be found, will revert when its merge time.

**Comment by lithomas1** [comment]
This is ready now.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @lithomas1

**Pull request merged**